olive/model.py

```diff
@@ -247,6 +219,45 @@ def get_execution_providers(self, device: Device):
 
         return eps if eps else available_providers
 
+    def get_io_config(self):
+        if self.io_config:
```

Comment:
Could you help guide me to the usage of io_config or give a short introduction about this new concept in OnnxModel?

Reply:
Added a docstring for the method.